### ISSUE TITLE
Playlist definitiva

### ISSUE THREAD

**Issue by diegofrutuoso** [CloudRadioProject/BugReport#23]
Implementar playlist definitiva

========================================

Primeiro habilitar playlist para dentro das estações, o dono/administrador da estação cria as playlists baseado somente em seu próprio conteúdo. Amarração será feita com os próprios dados da estação.
Habilitar nova aba ao lado do embed contendo essa informação, deverá mostrar somente a thumb da playlist e uma opção "somente para dono e adm" de criar nova playlist.
Ao clicar numa playlist ele mostra os itens que já existem, um botão de reproduzir playlist e "somente para dono e adm" um botão de "Adicionar episódios".
Ao clicar em reproduzir playlist ele limpa a fila atual do player e começa a reproduzir do primeiro item.
Se clicado em algum item especifico da playlist, ele limpa a fila do player e adiciona todos os itens, porém começa a reproduzir do episódio que foi clicado.

Em segundo momento habilitar que o usuário comum crie playlists, essas que podem ter episódios de qualquer estação.

**Comment by lczanatta** [comment]
O sistema de PlayList esta implementada, só falta a integração com o player

**Comment by lczanatta** [comment]
Ajustes de designer realizados na pagina da playlist. Ainda esperando a integração com o novo player.

**Comment by AlanBessa** [comment]
Esperando vc para integrar. Se está pronto já sabe q terá que montar um array de strings. Tipo ['adda', 'dadsas', 'fhfghfg']

**Comment by lczanatta** [comment]
Sim. Mas vou esperar você puxar o player para o desenvolvimento substituindo o antigo. Dai eu integro. @AlanBessa preciso ver uma coisa com você em pvt!
